alchemy-web/docs/getting-started.md

```diff
@@ -122,80 +213,287 @@ After running your app, Alchemy creates a `.alchemy` directory to store state:
 .alchemy/
   my-first-app/         # app
     dev/                # stage
-      hello.txt.json  # resource
+      hello-worker.json # resource state
 ```
 
 State files help Alchemy determine whether to create, update, delete, or skip resources on subsequent runs.
 
-If you run the same script again without changes, you'll see no operations performed because the state hasn't changed.
-
 > [!NOTE]
 > Learn more about Alchemy state in [Concepts: State](./concepts/state.md)
 
-## Update our File
+## Generate wrangler.json for Local Development
+
+Add wrangler.json generation to your `alchemy.run.ts`:
+
+```typescript
+import { Worker, WranglerJson } from "alchemy/cloudflare";
 
-Let's now update our `alchemy.run.ts` script to change the content of the file:
+// ... existing code ...
 
-```ts
-const hello = await File("hello", {
-  // path: "./hello.txt",
-  path: "./hello-world.txt",
-  // content: "hello world"
-  content: "Hello, world!"
+const worker = await Worker("hello-worker", {
+  entrypoint: "./src/worker.ts",
+  url: true,
 });
+
+// Generate wrangler.json for local development
+await WranglerJson("wrangler.json", {
+  worker,
+});
+
+console.log(`Worker deployed at: ${worker.url}`);
+
+await app.finalize();
+```
+
+Run the script again to generate the `wrangler.json`:
+
+::: code-group
+
+```sh [bun]
+bun ./alchemy.run.ts
+```
+
+```sh [npm]
+npx tsx ./alchemy.run.ts
 ```
 
-Now, when we re-run the script, we'll see:
+```sh [pnpm]
+pnpm tsx ./alchemy.run.ts
 ```
-Update:  "my-first-app/dev/hello"
-Updated: "my-first-app/dev/hello"
+
+```sh [yarn]
+yarn tsx ./alchemy.run.ts
+```
+
+:::
+
+## Type-Safe Bindings
+
+One of Alchemy's key features is type-safe bindings that automatically infer types from your infrastructure configuration. Unlike traditional IaC tools, Alchemy doesn't generate types—it infers them directly from your TypeScript configuration.
```

Comment:
I think showing free safe bindings should happen before showing local development and wrangler json

Also, I think we should illustrate by example instead of introduce type safe bindings front loaded. 

Instead, after deploying the initial worker, then introduce bindings by creating a kv namespace and binding it. 

Then, as part of the step of using the binding at runtime, then introduce how we infer the type from the worker alchemy instance